### PR TITLE
[#1197] limit piechart categories

### PR DESCRIPTION
**Pull request by vitorbaptista**
The slice that groups the smaller values can't be clicked on, as it makes no
sense to drill down in multiple categories. Unfortunately, the way we detect if
the slice is the "Others" slice is by checking its name. Ideally, we would have
a flag in the "Others" slice, but that wasn't possible.

I also was unable to remove the pointer cursor on the "Others" slice, which is
confusing for the user. To do that, we need a stable class name, but the class
is based on the slice's label, which can change depending on the site language.
I'll open an issue for this.

Fixes openspending/openspending#1197

### CODE REVIEW

**Comment by vitorbaptista** [comment]
@akariv Just FYI, I haven't added the new translations because I don't have access to the spreadsheet.

**Comment by akariv** [comment]
@vitorbaptista weird, it says 'anyone with the link can edit'... anyway, iirc the spreadsheet is obsolete and we're now just using the JSON files.

File: app/front/scripts/directives/visualizations/piechart/template.html
Comment (akariv) [inline app/front/scripts/directives/visualizations/piechart/template.html:12]: where is this parameter defined?

**Comment by akariv** [comment]
Aside from one comment it lgtm

**Review by akariv — APPROVED** [review]
Aside from one comment

File: app/front/scripts/directives/visualizations/piechart/template.html
Comment (akariv) [reply app/front/scripts/directives/visualizations/piechart/template.html:12]: Ah, nm, found it.
Better to update the babbage.ui version dependency to make sure it uses the version that defines this parameter.

Comment (vitorbaptista) [reply app/front/scripts/directives/visualizations/piechart/template.html:12]: Sure, this PR is blocked on https://github.com/openspending/babbage.ui/pull/62 then.

Comment (vitorbaptista) [reply app/front/scripts/directives/visualizations/piechart/template.html:12]: @akariv I just merged https://github.com/openspending/babbage.ui/pull/62, now we need to release a new version. Do we have any document on how to do it, or shall I simply follow whatever was done before?

Comment (akariv) [reply app/front/scripts/directives/visualizations/piechart/template.html:12]: I guess the process would be to push new version to npm (I'll make you a maintainer)
Then update the dependencies in os-viewer and merge the PR.
You will be able to see the modifications in staging.openspending.org (it auto-deploys there).

**Comment by vitorbaptista** [comment]
@akariv Regarding the translations, I just add it to `en.json` and `es.json`. If we're not using the spreadsheet anymore, shall we remove https://github.com/openspending/os-viewer/blob/4efa8df2261e177800fbabc6bee40db05c1e4729/download_translations.py ?

**Comment by akariv** [comment]
Yes please (to both suggestions)

**Pull request merged**
